src/windows_native/descriptor/encoder.rs

```diff
@@ -1,7 +1,7 @@
-use crate::windows_native::descriptor::typedefs::{Caps, LinkCollectionNode};
-use crate::windows_native::descriptor::types::{ItemNodeType, Items, MainItemNode, MainItems};
-use crate::windows_native::error::{WinError, WinResult};
-use crate::windows_native::utils::PeakIterExt;
+use super::typedefs::{Caps, LinkCollectionNode};
+use super::types::{ItemNodeType, Items, MainItemNode, MainItems};
+use super::super::error::{WinError, WinResult};
```

Comment:
I am not really a fan of relative paths for imports, when its necessary to go up multiple levels.
Its much clearer with one glance from *where* the item was imported with the absolute path.

Reply:
I made this change to keep the backend more self-contained and not depend on the name it was imported under. My original idea was to do something like this:
```rust
#[cfg_attr(feature = "linux-native", path = "linux_native.rs")]
#[cfg_attr(feature = "windows-native", path = "windows_native/mod.rs")]
mod platform;
```
Which obviously broke those imports.

After changing my approach to `cfg_if` I decided to nevertheless keep it. I can revert the paths if you want.